docs/extensions/capabilities/styling.md

```diff
@@ -1,8 +1,8 @@
-# Styling
-Lens provides a set of global styles and UI components that can be used by any extension to preserve look and feel of the application. However, it’s always up to the developer whether to use them or provide completely different styles for the extension.
+# Styling an extension
+Lens provides a set of global styles and UI components that can be used by any extension to preserve look and feel of the application.
 
 ## Styling approach
```

Comment:
I would rename this section to "Layout" and move the new "Basic Styling" (which is mostly about layout) section into this "Layout" section. I would leave-out the fonts, though, since that's more theme than layout.

So maybe like this:
```markdown
## Layout

For layout tasks, Lens is using a combination of CSS Variables, and flex.box...
...
...However, feel free to use any styling technique or framework like [Emotion](https://github.com/emotion-js/emotion) or just plain CSS if you prefer.

### Layout Variables

(The content from "Basic Styling" excluding fonts.)
```

Then, we could have the Theme section, for colors and fonts (even though you're currently not changing fonts with the themes, but that's an implementation detail, IMO), where "Themable Colors" becomes "Theme Variables":

```markdown
## Layout

...(as above)...

## Themes

Lens is using 2 built-in themes located in `src/renderer/themes` folder each for light and dark color schemes. Active theme can be changed in the `Preferences` page...
...

### Theme Variables

After theme file gets parsed it provides list of theme-defined colors. Most of their values are different for light and dark themes. You can use them to preserve consistent view of extension with respect of selected theme...
...
Colors values are located inside `src/renderer/themes/lens-dark.json` and `src/renderer/themes/lens-light.json` files.
```

So the document outline would look like this:
```markdown
# Styling an extension

## Layout

### Layout Variables

## Themes

### Theme Variables

## Injected Styles

## CSS-in-JS
```